src/itest/resources/features/company-insolvency.feature

```diff
@@ -6,12 +6,39 @@ Feature: Process company insolvency information
     When I send PUT request with payload "<data>" file
     Then I should receive 200 status code
     And the expected result should match "<result>" file
+    And the CHS Kafka API is invoked successfully
 
     Examples:
       | data                             | result                                  |
       | case_type_compulsory_liquidation | case_type_compulsory_liquidation_output |
       | case_type_receivership           | case_type_receivership_output           |
 
+  Scenario Outline: Processing company insolvency information unsuccessfully
+
+    Given Insolvency data api service is running
+    When I send PUT request with raw payload "<data>" file
+    Then I should receive <response_code> status code
+    And the CHS Kafka API is not invoked
+    And nothing is persisted in the database
+
+    Examples:
+      | data                                             | response_code |
+      | bad_request_payload                              | 400           |
+      | internal_server_error_payload                    | 500           |
+
+  Scenario Outline: Processing company insolvency information unsuccessfully but saves to database
```

Comment:
Covers the scenario:
<img width="778" alt="image" src="https://user-images.githubusercontent.com/16381559/164451486-ef3acb4f-ec65-4310-b2bc-c7145516b335.png">